### PR TITLE
GH-148: lira eats error qa tweex - Diagnose more problems in shell scripts.

### PR DESCRIPTION
**Pull request by tbl3rd**
### Purpose
_Please explain the purpose of this PR and include links to any GitHub issues that it fixes:_

- Fix problems discovered in QA for https://broadinstitute.atlassian.net/browse/GH-148
---
### Changes
_Please list out what major changes were made in this PR to address the issue:_

- More diagnostics in the docker scripts.
---
### Review Instructions
_Please provide instructions about how should a reviewer test/verify the changes in this PR:_

- Anyone else would probably know better than I.
- Run against CAAS and a non-OAuth Cromwell maybe?
---
### PR Checklist
_Please ensure the following when opening a PR:_

- [ ] This PR added or updated tests.
- [x] This PR updated docstrings or documentation.
- [ ] This PR applied Python style guidelines, specifically followed [Google style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google) for this repo.
- [ ] This PR considered generalizability beyond our own use case.

---
### Follow-up Discussions
_Please append follow-up discussions and issues during the review process below:_

- No follow-up discussions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,13 +72,23 @@ You can run Lira in docker or a Python virtual environment.
...
    See an example at lira/test/data/config.json.
+
 4. Build the docker container: `bash build_docker.sh test`
-5. Run the docker container:
-   `bash run_docker.sh test /path/to/config.json [CAAS_KEY [PORT]]`
+
+5. Run the docker container as shown below.
+
+   ``` shell
+   bash run_docker.sh test /absolute/path/to/config.json
```
Comment (rexwangcc) [inline README.md:87]: if the `use_caas` is `true` in the config, this should be `bash run_docker.sh test /absolute/path/to/config.json /absolute/path/to/service_account.json`

**Review by rexwangcc — APPROVED** [review]
One minor comment, otherwise LGTM!

File: README.md (hunk 1 above)
Comment (tbl3rd) [reply README.md:87]: Not any more, I think.  See ...
https://github.com/HumanCellAtlas/lira/blob/tbl/GH-148-Lira-eats-error-qa-tweex/run_docker.sh#L54-L59

Comment (rexwangcc) [reply README.md:87]: AH, didn't notice that, thanks for pointing out!

Comment (tbl3rd) [reply README.md:87]: IWBNI we could do something similar for the virtualenv test procedure.

**Comment by tbl3rd** [comment]
I don't understand why the Integration Tests don't succeed.
(-I remember @rexwangcc  explained, but don't remember his explanation!-)

**Pull request merged**